code/figure3.jl

```diff
@@ -0,0 +1,44 @@
+include("seed.jl")
+
+# Simulation with specialist natural enemies with m=0.2
+sim1, params1 = simulation(50.0, 25.0, m=0.2, F=4.0, D=0.5, c=1.0, a=0.5, th = 0.0)
```

Comment:
Just pass the values which are different from the default. It looks like all are the same as the default as `m=0.2`, but this isn't clear at a glance.